Update packages.md (#177)
scarf-sh/docs: docs/packages.md

![New file package route](assets/pics/qs-file-packages/file-package-new-route.png)

## Event Collection Packages

> Event Collection Packages on Scarf are another flexible package type which can be used to collect event telemetry by fetching a URL or by importing event history. Event Collection Packages are an alias of File packages and share all the same traits. Common use cases include:

Reviewer: ```suggestion
Event Collection Packages on Scarf are another flexible package type purpose built for telemetry data in general. Event data can be sent to a public Scarf Gateway URL of your choice, or by bulk ingesting events through our authenticated API. Event Collection Packages are an alias of File packages and share all the same traits. Common use cases include:
```

- Sending custom telemetry or other events from your application

> - Importing log or event history from your hosted installations to Scarf

Reviewer: ```suggestion
- Importing historical event data from an external application into to Scarf
```

### Creating an Event Collection Package
1. Once signed in to Scarf, navigate to the home page.

2. Click plus icon in the navigation, then select New Package.
![Create a new package](assets/pics/qs-file-packages/create-new-package.png)

3. In the first drop-down click on the package type you would like to create. For this section you will click `Event Collection`.
![Create a package](assets/pics/qs-file-packages/create-file.png)

4. Select the package owner from the dropdown.
![Select package owner](assets/pics/qs-file-packages/file-package-select-owner.png)

5. Give your package a name.
![Name your package](assets/pics/qs-file-packages/file-package-name.png)

### Adding an Incoming URL

> This section explains what the Outgoing and Incoming URLs are and how to use a URL template.

Reviewer: We don't have outgoing URLs in this context actually. We also may want to call out that you don't _need_ to use public gateway URLs, eg if you are only going to use the bulk import API.

1.) Add the URL path where your events will be collected. This is the user visible endpoint your application will connect to for event submission. This setting while required is not relevant when submitting events via the [Event Import API](https://docs.scarf.sh/event-import/).

>  > Note: The Outgoing URL is the full URL to your asset on your HTTP/S hosting provider. It can be a URL template but if you use variables in your URL they need to also be used in your Incoming Path that define in the next step.

Reviewer: No outgoing URLs here

3.) Click **Submit**.

### Configuring Event Collection
Once an Event Collection package has been created, you are ready to collect [Custom telemetry](https://docs.scarf.sh/custom-telemetry/)

## Python Packages

Scarf Gateway configuration for a Python package entry has three main considerations:

- pip Command: This is the current pip command used to install your package. For packages on PyPI.org, this will be of the form pip install my-pkg and will include the --extra-index-url https://my-python-project-domain.com if your package is hosted elsewhere. This defines the location where the users will be redirected to when installing your package.
- Domain: This can be your own domain, or a Scarf-supplied domain, of the form <username>.gateway.scarf.sh. By default, your Scarf domain will be used if this field is left empty.
- Telemetry: This allows you to gather insights into how your package is used without collecting any personally identifiable information.

Installing Python packages via requirements.txt
Add the --extra-index-url option at the top of your requirements.txt:

> (''' --extra-index-url https://my-python-project-domain.com/simple/ ''')

Reviewer: Needs to be in a code block

my-pkg==0.0.1

NOTE: We have noticed indeterminate behavior in some versions of Pip that have resulted in the public registry being used for download regardless of the --extra-index-url addition.

> If you elect to use your own domain, you'll need to add a CNAME for that domain to <span style="color:gray;">https://gateway.scarf.sh</span>. Additionally we require you to verify your ownership of the domain by setting a TXT with a value that Scarf provides upon package creation. See your DNS provider's instructions for how to add CNAME and TXT records.

Reviewer: Use the ` character to do inline highlighting, no need to make html spans directly.

1. Once signed in to Scarf, navigate to the home page.

2. Click plus icon in the navigation, then select New Package.
![Create a new package](assets/pics/qs-file-packages/create-new-package.png)

3. In the first drop-down click on the package type you would like to create. For this section you will click `Event Collection`.
![Create a package](assets/pics/qs-file-packages/create-file.png)

4. Select the package owner from the dropdown.
![Select package owner](assets/pics/qs-file-packages/file-package-select-owner.png)

5. Give your package a name.
![Name your package](assets/pics/qs-file-packages/file-package-name.png)

### Adding an Incoming URL
This section explains what the Incoming URLs are and how to use a URL template for Event Collection. 

1.) Add the URL path where your events will be collected. This is the user visible endpoint your application will connect to for event submission. This setting while required is not relevant when submitting events via the [Event Import API](https://docs.scarf.sh/event-import/).
 > Note: You can use a URL template, but if you use variables in your URL they need to also be used in your Incoming Path that is defined in the next step.

> ![path where files are located](assets/pics/qs-file-packages/file-package-outgoing-url.png)

Reviewer: Wrong image?

3.) Click **Submit**.

### Configuring Event Collection
Once an Event Collection package has been created, you are ready to collect [Custom telemetry](https://docs.scarf.sh/custom-telemetry/)

## Python Packages

Scarf Gateway configuration for a Python package entry has three main considerations:

- pip Command: This is the current pip command used to install your package. For packages on PyPI.org, this will be of the form pip install my-pkg and will include the --extra-index-url https://my-python-project-domain.com if your package is hosted elsewhere. This defines the location where the users will be redirected to when installing your package.
- Domain: This can be your own domain, or a Scarf-supplied domain, of the form <username>.gateway.scarf.sh. By default, your Scarf domain will be used if this field is left empty.
- Telemetry: This allows you to gather insights into how your package is used without collecting any personally identifiable information.

Installing Python packages via requirements.txt
Add the --extra-index-url option at the top of your requirements.txt:

> ``` --extra-index-url https://my-python-project-domain.com/simple/ ```

Reviewer: wrong tick location
